contrib/verifybinaries/README.md

```diff
@@ -1,16 +1,5 @@
 ### Verify Binaries
 
-#### Preparation:
```

Comment:
27107: just a note: maybe we want to dashify `contrib/verifybinaries/README.md` someday?